Jenkinsfile

```diff
@@ -130,13 +130,9 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
             'BUILD_SMPE'      : (params.BUILD_SMPE ? 'yes' : ''),
             'KEEP_TEMP_FOLDER': (params.KEEP_TEMP_FOLDER ? 'yes' : '')
           ],
-          extraFiles          : (params.BUILD_SMPE ? 'zowe-smpe.zip,fmid.zip,ptf.zip,pd.htm,smpe-promote.tar,smpe-build-logs.pax.Z,rename-back.sh' : ''),
+          extraFiles          : (params.BUILD_SMPE ? 'zowe-smpe.zip,pd.htm,smpe-promote.tar,smpe-build-logs.pax.Z' : ''),
```

Comment:
we have an overlap with https://github.com/zowe/zowe-install-packaging/pull/1180 here. As mentioned in John's pull request, we must always place fmid.zip and ptf.zip in Artifactory to accommodate automated testing and redistribution of PTF via Shopz.  zowe-smpe.zip is not suitable as its content is variable (FMID in pre-GA builds and PTF in post-GA builds).

I have no problem with dropping rename-back.sh. It will simplify automated pickup of the fmid and ptf if their zip file has a fixed name.